<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config</class>
 <widget class="QDialog" name="config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/Config.png</normaloff>:/images/Config.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>84</width>
         <height>284</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Account</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/Account.png</normaloff>:/images/Account.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GIS</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/GIS.png</normaloff>:/images/GIS.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/Update.png</normaloff>:/images/Update.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageAccount">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>User</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="labelUser">
                <property name="text">
                 <string>&amp;Name:</string>
                </property>
                <property name="buddy">
                 <cstring>lineEditUser</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditUser"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelPassword">
                <property name="text">
                 <string>&amp;Password:</string>
                </property>
                <property name="buddy">
                 <cstring>lineEditPassword</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditPassword">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAccount">
              <property name="text">
               <string>Save &amp;Account</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageSite">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBoxBSC">
           <property name="title">
            <string>BSC</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="hLayoutIP">
              <item>
               <widget class="QLabel" name="labelIP">
                <property name="text">
                 <string>IP List</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="hSpacerIP">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonAdd">
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/images/Add.png</normaloff>:/images/Add.png</iconset>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonDelete">
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/images/Delete.png</normaloff>:/images/Delete.png</iconset>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetBSC">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>129</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
           <zorder></zorder>
           <zorder>listWidgetBSC</zorder>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxGIS">
           <property name="title">
            <string>GIS</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&amp;File:</string>
              </property>
              <property name="buddy">
               <cstring>lineEditGIS</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditGIS"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonBrowse">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageInterval">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Update Interval</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDial" name="dialInterval">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelTimer">
                <property name="text">
                 <string>Minute(s):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="numDigits">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
                <property name="intValue" stdset="0">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Log</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&amp;Number of Logs per Site:</string>
              </property>
              <property name="buddy">
               <cstring>spinBoxLog</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxLog">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>config</receiver>
   <slot>addIP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonDelete</sender>
   <signal>clicked()</signal>
   <receiver>config</receiver>
   <slot>deleteIP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonBrowse</sender>
   <signal>clicked()</signal>
   <receiver>config</receiver>
   <slot>browseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialInterval</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addIP()</slot>
  <slot>deleteIP()</slot>
  <slot>browseFile()</slot>
 </slots>
</ui>
